.github/workflows/e2e.yml

```diff
@@ -0,0 +1,52 @@
+name: test e2e
+on: [push, pull_request]
+jobs:
+  test-e2e:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]  
+
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.11'
+
+      - name: Set env
+        shell: bash
+        run: |
+          echo "::set-env name=GOPATH::${{ github.workspace }}"
+          echo "::add-path::${{ github.workspace }}/bin"
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: src/github.com/kubernetes-sigs/cri-tools
+
+      - name: Run e2e test
+        shell: bash
+        run: |
+          sudo apt-get update &&\
+          sudo apt-get install -y libseccomp-dev
+        
+          VERSION=v0.8.2 &&\
+          sudo mkdir -p /opt/cni/bin &&\
+          sudo wget -qO- https://github.com/containernetworking/plugins/releases/download/$VERSION/cni-plugins-linux-amd64-$VERSION.tgz \
+              | sudo tar xfz - -C /opt/cni/bin &&\
+          ls -lah /opt/cni/bin
+
+          VERSION=v1.0.0-rc8 &&\
```

Comment:
Can we use the latest release?

Reply:
Sure, updated cni plugin version as well.